components/constituency_lookup/ConstituencyLookup.tsx

```diff
@@ -88,41 +88,41 @@ const PostcodeLookup = () => {
     // Update the stored / displayed postcode
     setFormState({ ...formState, postcode: userPostcode });
 
-    const trimmedPostcode = userPostcode.trim();
+    const normalizedPostcode = userPostcode
+      .replaceAll(sanitizePostcode, "")
+      .toUpperCase();
```

Comment:
As we're now repeating this in client & server, might make sense to create a "normalizePostcode" method in the postcode utils.

Reply:
Yeah good idea, the UI could get in a very weird state if these don't match.